source/_posts/2016-10-14-backing-up-and-restoring-a-postresql-database.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Backing Up and Restoring a PostgreSQL Database"
```

Comment:
@codalicious keyword-wise... When you are trying to find a post like this what keywords would you use?

Personally, I would search for something like `backup restore postgresql`... I doubt that I would use `backing up` and `restoring`. Of course - I might be wrong. Check your browser history and use whatever you used to find info on this subject.